src/pilot/discover_nodes/discover_nodes.py

```diff
@@ -501,15 +501,8 @@ def is_idrac(client):
         # Ensure the libraries' loggers are re-enabled.
         requests_logger.disabled = False
         dracclient_wsman_logger.disabled = False
-
-    if doc is None:
-        return False
-
-    return dracclient.utils.get_wsman_resource_attr(
-        doc,
-        DCIM_iDRACCardView,
-        'DeviceDescription') == 'iDRAC'
-
+    
+    return is_ready
```

Comment:
The server is an iDRAC if the is_idrac_ready call returns true or false.  As a result we don't want to return the value of is_ready, we just want to return a hard-coded true if no exception occurred.